rt/test/TESTOPR2.MLC

```diff
@@ -0,0 +1,1443 @@
+
+&B2D     SETC  B2D('')
+         MNOTE 'B2D('''')=&B2D'
+         AIF   ('&B2D' NE '+0').B2ERR
+&B2D     SETC  B2D('00010010001')
+         MNOTE 'B2D(''00010010001'')=&B2D'
+         AIF   ('&B2D' NE '+145').B2ERR
+&B2D     SETC  B2D('11110001')
+         MNOTE 'B2D(''11110001'')=&B2D'
+         AIF   ('&B2D' NE '+241').B2ERR
+&B2D     SETC  B2D('01111111111111111111111111111111')
+         MNOTE 'B2D(''01111111111111111111111111111111'')=&B2D'
+         AIF   ('&B2D' NE '+2147483647').B2ERR
+&B2D     SETC  B2D('11111111111111111111111111110001')
+         MNOTE 'B2D(''11111111111111111111111111110001'')=&B2D'
+         AIF   ('&B2D' NE '-15').B2ERR
+*----------------------------------------------------------------------
+         MNOTE '********** End   B2D tests **********'
+*======================================================================
+         MNOTE '********** Begin B2X tests **********'
+*----------------------------------------------------------------------
+*        HLASM LR examples
+
+&B2X     SETC  B2X('')
+         MNOTE 'B2X('''')=>&B2X<'
+         AIF   (K'&B2X NE 0).B2ERR
+&B2X     SETC  B2X('00000')
+         MNOTE 'B2X(''00000'')=&B2X'
+         AIF   ('&B2X' NE '00').B2ERR
+&B2X     SETC  B2X('0000010010001')
+         MNOTE 'B2X(''0000010010001'')=&B2X'
+         AIF   ('&B2X' NE '0091').B2ERR
+&B2X     SETC  B2X('11110001')
+         MNOTE 'B2X(''11110001'')=&B2X'
+         AIF   ('&B2X' NE 'F1').B2ERR
+&B2X     SETC  B2X('1111110001')
+         MNOTE 'B2X(''1111110001'')=&B2X'
+         AIF   ('&B2X' NE '3F1').B2ERR
+*----------------------------------------------------------------------
+         MNOTE '********** End   B2X tests **********'
+*======================================================================
+         MNOTE '********** Begin C2A tests **********'
+*----------------------------------------------------------------------
+*        HLASM LR examples
+
+&C2A     SETA  C2A('')
+         MNOTE 'C2A('''')=&C2A'
+         AIF   (&C2A NE 0).C2ERR
+&C2A     SETA  C2A('+')
+         MNOTE 'C2A(''+'')=&C2A'
+         AIF   (&C2A NE 78).C2ERR
+&C2A     SETA  C2A('1')
+         MNOTE 'C2A(''1'')=&C2A'
+         AIF   (&C2A NE 241).C2ERR
+&C2A     SETA  C2A('0000')
+&SC2A    SETC  SIGNED(&C2A)
+         MNOTE 'C2A(''0000'')=&SC2A'
+         AIF   (&C2A NE -252645136).C2ERR
+         AIF   ('&SC2A' NE '-252645136').C2ERR
+*----------------------------------------------------------------------
+         MNOTE '********** End   C2A tests **********'
+*======================================================================
+         MNOTE '********** Begin C2B tests **********'
+*----------------------------------------------------------------------
+*        HLASM LR examples
+
+&C2B     SETC  C2B('')
+         MNOTE 'C2B('''')=>&C2B<'
+         AIF   (K'&C2B NE 0).C2ERR
+***********************************************************************
+*        TODO how to type the null character?
```

Comment:
If there is anything left to do: please complete the action and remove the remark. 
Alternatively, open new issue to cover the todo item and equally remove the remark.